lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -222,16 +240,28 @@ def info
 
       private
 
-        def pred_id_from_ldpath
+        def id_predicates_from_ldpath
           # prefix example: { skos: 'http://www.w3.org/2004/02/skos/core#' }
           # ldpath example: 'skos:id :: xsd:string'
```

Comment:
We talked about adding some examples or documentation here to show what having multiple paths would be like.